src/@next/components/organisms/PaymentGatewaysList/types.ts

```diff
@@ -39,10 +39,7 @@ export interface IProps {
     token?: string,
     cardData?: ICardData
   ) => void;
-  submitPayment: (data: {
-    confirmationData: any;
-    confirmationNeeded: boolean;
-  }) => Promise<any>;
+  submitPayment: (data?: object) => Promise<any>;
```

Comment:
Isn't there a specific type we could use for the returned Promise?

Reply:
And if there isn't, it should be of type `unknown`, not `any`.

Reply:
Fixed